examples/vector_databases/cassandra_astradb/Philosophical_Quotes_CQL.ipynb

```diff
@@ -0,0 +1,1353 @@
+{
```

Comment:
<p>minor typo: <code>This notebook can run either as a Colab notebook or <s>with</s> as a regular Jupyter notebook.</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openai/openai-cookbook/pull/655/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>